src/displayProjects.js

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable */
+import { Storage } from './storage';
+import { ProjectDOMElement } from './projectDom';
+
+const showProjects = () => {
+  const projectDisplay = document.querySelector('.project-display-container');
+  const project = Storage.getProjects();
+
+  project.forEach((element, index) => {
+    const project = new ProjectDOMElement(element.title, index);
+
+    projectDisplay.appendChild(project.div);
+  });
+};
+
+export { showProjects };
```

Comment:
If you're only exporting one object/function in a file, lest add the default flag, this issue repeats in your project, let's fix it. example: `export default showProject`